css/css-ui-3/text-overflow-015.html

```diff
@@ -0,0 +1,29 @@
+﻿<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: ellipsis after graphical transformations</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#ellipsing-details">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem">
+<meta name="assert" content="Ellipsing occurs after relative positioning and other graphical transformations">
```

Comment:
Actually, that was a mistake. The spec calls for "and other graphical transformations", but css transforms don't apply to (non-atomic) inlines, and you can only get the ellipsis in a non atomc inline. So the the situation it hints at does not exist, and there's nothing to test for. Reverted.